Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM ubuntu:wily
+
+COPY loopback /opt/loopback
```

Comment:
I left out the ENTRYOPOINT on purpose for now to allow users doing nasty tricks with nsenter since Docker version prior to 1.10 don't support rootfs mount propagation.